bp_common/test/src/perch/start.S

```diff
@@ -5,7 +5,7 @@ _start:
 /* setup stack pointers. Stacks start at 0x8FFF_CFFF */
 /* We then subtract off 8K*coreID. The top 4K is for the core emulation stack
  * the lower 4K is for the program TODO: Maybe larger stack value? */
-    li   sp, 0x8FFFCFF0
+    li   sp, 0x80010000
```

Comment:
Seems like a pretty small stack.  Have you tried running e.g. SPEC VPR with this?

Maybe we should make it configurable?

Reply:
No, I don't have access to that repo. How does the configuration work though? Because I think it should be recompiled every time we change the stack size.

Reply:
What do you mean by the configuration?

But yes, this way we need to recompile libperch for each different stack size.  I think some way of the firmware querying the stack size of the program is ideal.  I guess it should use the same mechanism as getargs? (which we haven't implemented yet)